test/dotnet-watch.Tests/CommandLine/CommandLineOptionsTests.cs

```diff
@@ -127,7 +124,7 @@ public void WatchOptions_NotPassedThrough_BeforeCommand(
         {
             var options = VerifyOptions(before ? [option, "test"] : ["test", option]);
             Assert.Equal("test", options.Command);
-            AssertEx.SequenceEqual(option == "--non-interactive" ? [] : [InteractiveOption], options.CommandArguments);
+            AssertEx.SequenceEqual(option == "--non-interactive" ? [] : [], options.CommandArguments);
```

Comment:
Uhh...
> option == "--non-interactive" ? [] : []

So, something's not right there.

Edit: Oh, I see. You removed `InteractiveOption` from everything, so this ternary is no longer needed.